frontend/src/e2e-playwright/jest-environment.js

```diff
@@ -20,7 +20,9 @@ class PlaywrightEnvironment extends NodeEnvironment {
       const specName = event.test.name.replace(/\W/g, '-')
 
       const namePrefix = `${parentName}_${specName}`
-      await this.global.evaka?.takeScreenshots(namePrefix)
+      await this.global.evaka?.saveScreenshotsAndVideos(namePrefix)
+    } else if (event.name === 'test_fn_success') {
```

Comment:
Where this `test_fn_success` event name from?

Reply:
From `jest-circus` test runner, like the `test_fn_failure` above. The library's documentation is pretty sparse, but seems to work 🤷